components/eam/src/physics/cam/shoc_intr.F90

```diff
@@ -580,7 +583,7 @@ subroutine shoc_tend_e3sm( &
    real(r8) :: ice_cloud_frac(pcols,pver)          ! ice number aware cloud fraction, 0 or 1
    real(r8) :: precipitating_ice_frac(pcols,pver)        ! precipitating ice fraction, 0 or 1
    real(r8) :: liq_cloud_frac(pcols,pver)     
-   real(r8) :: tot_cloud_frac(pcols,pver)         
+   !real(r8) :: tot_cloud_frac(pcols,pver)         
```

Comment:
I'm in favor of removing this line rather than commenting it out.

Reply:
I agree with Peter.